migrations/local_ledger/migration.go

```diff
@@ -61,6 +61,10 @@ func RunMigrationSimple(round uint64, opts *idb.IndexerDbOptions) error {
 	if err != nil {
 		return fmt.Errorf("RunMigration() err: %w", err)
 	}
+	// ledger and db states are in sync
+	if proc.NextRoundToProcess()-1 == round {
+		return nil
```

Comment:
Nit: You can just return err here since it should be nil if we haven't errored out already.